specs/eventing/control-plane/channel-lifecycle/services.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: serving.knative.dev/v1
```

Comment:
I would use a plain k8s service, to minimize the number of external dependencies and also but default Knative services scale down to 0, making it harder to observe events in the logs (more time-sensitive)

Reply:
Since the ping sources send events every 1 minute it should not happen until they are deleted. I changed to use the events display image so it is aligned with everything else in the tutorials and its also an internal dependency.